### ISSUE TITLE
Roberta multiple choice is failing deterministically on Grayskull

### ISSUE THREAD

**Issue by tt-rkim** [tenstorrent-metal/tt-metal#5487]
`models/experimental/roberta/tests/test_roberta_for_multiple_choice.py::test_roberta_for_multiple_choice` is failing deterministically on Grayskull. We need to look into it

cc: @TT-billteng

**Comment by boris-drazic** [comment]
This test started failing after commit [4b39362](https://github.com/tenstorrent-metal/tt-metal/commit/4b39362cefd996597df7bf192c65b8738f8e8e25).

Previously computed logits were PyTorch [0.1566, 0.1597] and TT [0.157227, 0.161133]. After commit we have PyTorch [-0.0451, -0.0456] and TT [-0.0371094, -0.0390625]. Values still match but with slightly higher difference. PR #5493 to raise ATOL from 0.008 to 0.0081 will make this test pass.

**Comment by TT-billteng** [comment]
@vtangTT can you confirm if this is true or not

**Comment by tt-rkim** [comment]
@boris-drazic do we no longer see this in scheduled pipelines?

**Comment by boris-drazic** [comment]
No longer seeing this failure on pipelines.
